.github/config/doubler.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
why is this in `.github` while `test_process.py` remains in `.ci`?
I guess we keep general stuff in `.ci` and move only the github-specific things under `.github`?

Reply:
Everything that is under `.github` now is GitHub specific though. None of it is being used elsewhere. I kept the `test_` scripts in `.ci` because those are still being run by Travis/Jenkins. Once we are happy with the current changes, I will remove Travis and trim down the requirements for Jenkins. This will allow me to clean up those files significantly. What is used by both I will keep in `.ci`

Reply:
Sounds good.